### PR TITLE
remove underscore prefix from property names

### PR DESCRIPTION
**Pull request by morrme**
Signed-off-by: morrme <morrme@users.noreply.github.com>

### Description

Removes underscore from property names
See #16278 

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

File: libraries/classes/Bookmark.php [hunk 1]
```diff
@@ -25,31 +25,31 @@ class Bookmark
...
      */
-    private $_database;
+    private $database;
     /**
      * The user to whom the bookmark belongs, empty for public bookmarks
      *
      * @var string
      */
-    private $_user;
+    private $user;
```
Comment (mauriciofauth) [inline libraries/classes/Bookmark.php:40]: The property `$user` already exists.
https://github.com/phpmyadmin/phpmyadmin/blob/bcf21c4081232df1bb77a076a3ff551ee7d7b40c/libraries/classes/Bookmark.php#L57-L62

**Review by MauricioFauth — CHANGES_REQUESTED** [review]

**Comment by mauriciofauth** [comment]
Hello. Thanks for your contribution!

We require a full real name for legal compliance reasons. Could you use your full name in the signature tag?
For example: `Signed-off-by: Jane Developer <jane@example.org>`

See [Developer's Certificate of Origin](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO#L39)

**Comment by morrme** [comment]
@mauriciofauth Please see the note I wrote you aout $user and let me know your decision

**Comment by mauriciofauth** [comment]
> @mauriciofauth Please see the note I wrote you aout $user and let me know your decision

Sorry, but I don't see where your note is.

**Comment by morrme** [comment]
Please see the review line above your comment. What would you like to change $user to?

**Comment by morrme** [comment]
@mauriciofauth  Any update here? Is there a variable name you'd like to change $_user to so that it does not conflict with $user?

**Comment by morrme** [comment]
bk
@mauriciofauth here is my original comment

File: libraries/classes/Bookmark.php (hunk 1 above)
Comment (mauriciofauth) [reply libraries/classes/Bookmark.php:40]: You can use `$bookmarkUser` here.

**Comment by mauriciofauth** [comment]
@morrme I didn't saw your comment because it was still with the _Pending_ status. I can use `$bookmarkUser` instead of `$_user`.

**Pull request closed without merging**